### PR TITLE
Add a !choose (alias !pick) command

### LINKED ISSUE #12: Choose command
The !choose command (alias !pick) should take a list of options as input and return on of them at random.

Eg: "!choose go to sleep, eat more, programming" should return either "go to sleep", "eat more" or "programming" to the user.

### PR DESCRIPTION
**Pull request by pralkarz**
As mentioned in the commit message, the parsing is somewhat hacky, but I didn't know how to solve that better. Regular splitting wouldn't work as then the whitespace was removed. I'm open for suggestions on how I might improve it!

Closes #12

### CODE REVIEW

File: commands/choose.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports.execute = async (client, message, args) => {
+	// Convert everything to string so the args can be parsed more easily
+	for (let i = 0; i < args.length - 1; i++) {
+		args[i] = args[i].toString();
+	}
```
Comment (Moesgaarda) [inline commands/choose.js]: Is this needed? Which type does args have before converting them? (Hint, you can use [typeOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) to check this)

File: commands/choose.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+	global.Math = mockMath;
+
+	// Act
+	const responseMessage = await choose.execute(null, message, ['Knights', 'of', 'Academia,', 'Academia', 'of', 'Knights']);
+
+	// Assert
+	const expectedResponse = 'Horace says... Knights of Academia!';
+	expect(responseMessage).toBe(expectedResponse);
+});
```
Comment (Moesgaarda) [inline commands/choose.test.js]: Does test the full functionality of the command? Can it ever pick the second or possibly object? What about using choose with no args?

**Review by Tekrus — CHANGES_REQUESTED** [review]
Thanks for your contribution! Here's some feedback on how you can improve it a little as you wished for. 👋

**Comment by ziebam** [comment]
I've done what I can, although I'm still open to feedback! I'm mostly not sure if the regex parsing is correct there or if I should seek a better way.

**Review by Dendrobyte — APPROVED** [review]
!choose Approve,Comment,Request Changes

Horace says... Approve!

**Review by erik-ahlburg-schmidt — APPROVED** [review]

**Review by Tekrus — APPROVED** [review]

**Comment by Moesgaarda** [comment]
Triple approved!

**Pull request merged**